Wattpad2epub.py

```diff
@@ -86,7 +87,23 @@
 print("saved "+ story_name+".html")
 print("Generating Epub...")
 
+cover_image = story_name+".jpg"
+urllib.request.urlretrieve(cover, cover_image)
+
+xmlfile = open('details.xml', 'r')
+body = xmlfile.read()
+xmlfile.close()
+
+xmlfile = open('details.xml', 'w')
+xmlfile.write(body.format(title=story_name, author=author))
+xmlfile.close()
+
 #Using Pypandoc to convert html to epub
-output = pypandoc.convert_file(story_name+".html", 'epub3', outputfile=story_name+".epub", extra_args=['--epub-chapter-level=2'])
+output = pypandoc.convert_file(story_name+".html", 'epub3', outputfile=story_name+".epub", extra_args=['--epub-chapter-level=2', '--epub-metadata=details.xml', '--epub-cover-image={0}'.format(cover_image)])
```

Comment:
This is good, we will just need the `--epub-cover-image` attribute, if we are going to implement as above.